ajax/libs/videojs-ima/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "videojs-ima",
+  "filename": "videojs.ima.min.js",
+  "description": "IMA SDK Plugin for Video.js ",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Google Inc."
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/googleads/videojs-ima"
+  },
+  "keywords": [
+    "Video.js",
+    "plugin",
+    "IMA",
+    "SDK"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/googleads/videojs-ima.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "videojs.ima*"
```

Comment:
Is there any reason that why don't use `**/*`?
I think use `**/*` is more flexible when it change the file name.